ModulemdTranslationHelpers/Utils.py

```diff
@@ -1,14 +1,111 @@
 import sys
+import logging
+import gi
+import requests
+import koji
 from babel.messages import Catalog, pofile
 from datetime import datetime
 from collections import defaultdict
 
-try:
-    import gi
-    gi.require_version('Modulemd', '2.0')
-    from gi.repository import Modulemd
-except ImportError:
-    sys.exit(77)
+gi.require_version('Modulemd', '2.0')  # noqa
+from gi.repository import Modulemd
+
+
+class KojiMock:
+
+    def __init__(self, url):
+        self.session = koji.ClientSession(baseurl=url)
```

Comment:
The whole point of creating a mock object is to *not* actually use `koji.ClientSession`. You instead create an object that simulates it.